### PR TITLE
Add support for FDB 630

### PR DESCRIPTION
**Pull request by PierreZ**
This PR is adding support for FoundationDB version 6.3.X.

It has been tested locally with 4k+ passing correctness tests for 6.3, which is far better than the current state of the repo.

It also includes a fix on the bindingTester that will fix 99% of the failing scheduled correctness tests.

### CODE REVIEW

**Comment by PierreZ** [comment]
We should restart the CI when foundationdb.org will be up

**Comment by PierreZ** [comment]
Passed 3k correctness tests on fdb 630 🎉
Waiting for a review before merging.

**Review by Speedy37 — APPROVED** [review]

**Comment by PierreZ** [comment]
Hey thanks for the approval @Speedy37 :+1:

**Pull request merged**
